tests/deploy/base/mocks.yaml

```diff
@@ -35,15 +35,15 @@ data:
           include:
             match_type: strict
             resource_attributes:
-              - Key: sw.k8s.log.type
-                Value: event
+              - key: sw.k8s.log.type
```

Comment:
Since 0.71 the OTEL Collector has more strict configuration checks and these (uppercase) were no longer valid.